ledger/participant-state/kvutils/tools/codec-benchmark/src/benchmark/scala/com/daml/lf/benchmark/package.scala

```diff
@@ -89,6 +89,6 @@ package object benchmark {
     encodeTransaction(NidEncoder, CidEncoder, transaction)
 
   private def encode(value: DecodedValue): EncodeResult[EncodedValue] =
-    encodeVersionedValue(CidEncoder, value, ValueVersion.DevOutputVersions)
+    encodeVersionedValue(CidEncoder, TransactionVersion.VDev, value)
```

Comment:
Probably didn't need my input, but the changes to the benchmark looks good. 👍🏻

Reply:
Thanks a lot ;)